docs/other/fontobject.rst

```diff
@@ -35,6 +88,23 @@ String; read-only.
 
 ----
 
+.. _FontObject.familyPrefix:
+
+FontObject.familyPrefix
+*********************************************
+
+``app.fonts.fontsWithDefaultDesignAxes()[0].familyPrefix``
+
+**Description**
+
+The family prefix of the variable font.
```

Comment:
For example, the family of the PostScript name "SFPro-Bold" is "SFPro".

Also, will return undefined for non variable fonts.